src/runtime/rpc/rpc_host_port.cpp

```diff
@@ -80,6 +81,7 @@ host_port::host_port(rpc_address addr)
     } break;
     case HOST_TYPE_GROUP: {
         _group_host_port = new rpc_group_host_port(addr.group_address());
```

Comment:
Are there some special reasons why a raw pointer should be used here ? Consider using the object instead ?

If a pointer is needed, consider `std::unique_ptr`, or use `dsn::ref_ptr` to manage a `ref_counter` object at least ?

Reply:
`ref_counter` is super class of `rpc_group_host_port`.